docs/migrations/web/MIGRATION-v3.md

```diff
@@ -178,6 +193,91 @@ Instead of using `.Button--square`, use a `.Button--symmetrical`.
 
 - `.Button--square` → `.Button--symmetrical`
 
+### Button: The `inverted` Modifier Removed
+
+Button `inverted` modifier was removed. Use themes to switch the button colors.
+
+### Field Group: Alignment
+
+Field Group component children are not aligned to `start` except when the `FieldGroup--fluid` modifier is used.
+
+### Header: The `inverted` Variant Removed
+
+Header `inverted` variant was removed. Instead, the `primary` variant was added to be further adjusted with themes.
+
+### Partner Logo: Stabilized
+
+Partner Logo component was stabilized.
+
+#### Migration Guide
+
+Rename all usages of `UNSTABLE_PartnerLogo` to `PartnerLogo`.
+
+- `<div class="PartnerLogo PartnerLogo--small PartnerLogo--safeArea">` → `<div class="PartnerLogo PartnerLogo--small PartnerLogo--safeArea">`
+
+### Pill: Renamed and Removed Variants
+
+Pill component variants `primary`, `secondary`, `tertiary`, `inverted` and `unselected` were removed.
+Instead, the `neutral` variant was added. The current list of variants is
+
+- `neutral`
+- `danger`
+- `informative`
+- `success`
+- `warning`
+
+#### Migration Guide
+
+Use the `neutral` or emotion variants. If you need other variants, please, let us know.
+
+### Product Logo: The `inverted` Modifier Removed
+
+Product Logo `inverted` modifier was removed. Use themes to switch the logo colors.
+
+### Product Logo: Stabilized
+
+Product Logo component was stabilized.
+
+#### Migration Guide
+
+Rename all usages of `UNSTABLE_ProductLogo` to `ProductLogo`.
+
+- `<div class="UNSTABLE_ProductLogo">` → `<div class="ProductLogo">`
+
+### TextField: CSS Variables Renamed
+
+TextField CSS variable `--width` was renamed to `--text-field-input-width`.
+
+#### Migration Guide
+
+Replace all usages of `--width` with `--text-field-input-width` in TextField component.
+
+- `<input class="TextField__input" style="--width: 4em;" />` → `<input class="TextField__input" style="--text-field-input-width: 4em;" />`
+
+### Toast: The `inverted` Variant Removed
+
+Toast `inverted` variant was removed. Use the `neutral` variant instead.
+
+#### Migration Guide
+
+Replace all usages of `ToastBar--inverted` with `ToastBar--neutral`.
+
+- `<div class="ToastBar ToastBar--inverted">` → `<div class="ToastBar ToastBar--neutral">`
+
+### Toast: Link and Close Button
+
+Toast component now has its own close button and link.
```

Comment:
```suggestion
The Toast component now has its own close button and link.
```

I think there should always be the definite article in the form of "the X component"…